### DISCUSSION TITLE
Bandwidth config setting doesn't seem to do much

### DISCUSSION THREAD

**Discussion by mdehrlich** [GerryDazoo/Slinger#501] [General]
I ran several configs to test the resulting communication data-rate at the player.  Resolution and framerate will change the bandwidth.  The bandwidth setting doesn't do anything.

**Comment by robosmith** [comment]
IF it's like SlingPlayer app, I believe bandwidth only has an effect when resolution is set to auto.

**Reply by RocketAlf** [reply]
Have had the same experience as mdehrlich. Possibly a stupid question (?) ... Which value of resolution stands for " resolution is set to auto"?

**Reply by mdehrlich** [reply]
There is no auto mode.  Auto requires a player to track the actual video reception rate, and asking the slingbox to modify higher or lower resolution.  Maybe there must be a slingbox command to change resolution without stopping and restarting the stream, but slinger does not have that built in.  Also the player needs to be able to handle the change mid-stream.  Maybe someone could decompile slingplayer to understand how it works?  I say just be happy we have slinger.
